sssd_test_framework/utils/adcli.py

```diff
@@ -85,9 +85,9 @@ def join(
         :type domain: str
         :param args: Additional arguments, defaults to None
         :type args: list[str] | None, optional
-        :param password: Password
+        :param password: Password.
```

Comment:
This fixes should go to different commit.

Reply:
dropped it.